build.module

```diff
@@ -5,7 +5,7 @@
       "basic": ["assert", "dns", "http", "net", "stream", "testdriver"],
       "extended": {
         "linux": ["adc", "ble", "dgram", "gpio", "i2c", "pwm", "spi", "uart"],
-        "nuttx": ["adc", "dgram", "gpio", "i2c", "pwm", "stm32f4dis", "uart"],
+        "nuttx": ["adc", "dgram", "gpio", "i2c", "pwm", "spi", "stm32f4dis", "uart"],
         "darwin": [],
         "tizen": ["adc", "ble", "dgram", "gpio", "i2c", "pwm", "spi", "uart", "https"],
         "tizenrt": ["adc", "dgram", "gpio", "i2c", "pwm", "uart"]
```

Comment:
What about tizenrt? Shouldn't we add SPI to it too?

Reply:
I updated it.